internal/zinc-compile-core/src/main/scala/sbt/internal/inc/AnalyzingCompiler.scala

```diff
@@ -83,14 +83,15 @@ final class AnalyzingCompiler(
     val progress = if (progressOpt.isPresent) progressOpt.get else IgnoreProgress
     val cp = classpath.map(converter.toPath)
     val arguments = compArgs.makeArguments(Nil, cp, options).toArray
+    val loader = getLoader(log)
```

Comment:
```suggestion
    // hold reference to avoid the class loader getting collected by G1GC sbt/zinc#914
    val loader = getLoader(log)
```

Reply:
It might be worth adding a comment there to avoid future refactoring.

Reply:
👍